containers/unit-tests/setup.sh

```diff
@@ -43,6 +43,7 @@ dependencies="\
     pkg-config \
     python3 \
     python3-mypy \
+    python3-pytest-asyncio \
```

Comment:
Can we configure pytest to error out for stuff like this instead of silently passing?

Reply:
I'll add that to my todo list .. that, or file a bug that it should just happen. The current behaviour is really not okay.

Reply:
I've done a bit of research.

See https://github.com/pytest-dev/pytest/issues/5023 for the suggested solution (`--strict-markers`) as well as links to lots of other discussion about why it's not so easy.  I'll do a patch to add `--strict-markers`.